### PR TITLE
Initial styling for form

### LINKED ISSUE #15: Feature - Forms 4.0.0-beta
**Is your feature request related to a problem? Please describe.**
Based on bootstrap 4 documentation

This is what need to be done to complete forms task in #11

https://getbootstrap.com/docs/4.3/components/forms/

Master:
http://d1kybmg72qo0dt.cloudfront.net/build/global/branch/master/www/index.html

- [x] Default Input, E,g Font, font-weight, padding, border-radius
- [x] ~~Select~~
- [x] textarea
- [x] States, e.g focus
- [x] Form-control
- [x] ~~File-input~~
- [x] Sizing
- [x] Readonly
- [x] ~~Readonly plain text~~
- [x] ~~Range inputs~~
- [x] ~~Checkbox Radio buttons default~~
- [x] ~~Checkbox / radio Stacked~~
- [x] ~~Checkbox / radio  disabled~~
- [x] ~~Checkbox / radio  Inline~~
- [x] ~~Checkbox / radio  without labels~~
- [x] Form-groups
- [x] ~~Form-grid~~
- [x] ~~Form-row~~
- [x] ~~Form horizontal~~
- [x] ~~Form horizontal label sizing~~
- [x] ~~Form Column sizing~~
- [x] ~~Form Auto-sizing~~
- [x] ~~Form Inline~~
- [x] ~~Custom form-control~~
- [x] ~~Help-text~~
- [x] ~~Help-text small/span~~
- [x] ~~Disabled forms~~
- [x] Validation Custom-styles
- [x] ~~Validation Browser~~
- [x] ~~Validation Server-side~~
- [x] ~~Validation Tooltip~~
- [x] ~~Custom checkbox/radio button~~
- [x] ~~Custom Switches~~
- [x] ~~Custom Select~~
- [x] ~~Custom Range~~
- [x] ~~Custom File upload~~

### PR DESCRIPTION
**Pull request by helloimela**
- Added form styling
- Added variable for border color in table and form

**Solving issue**</br>
Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)
Fixes: #15

### CODE REVIEW

File: src/styles/_variables.scss [hunk 1]
```diff
@@ -110,9 +114,19 @@ $h4-font-size-article:        $font-size-article * 1.33;
...
 
+$border-color:                $light;
+$table-border-color:          $border-color;
+$input-border-color:          $border-color;
+
+$input-padding-x:             1.2rem;
+$input-padding-y:             1rem;
+$input-padding-y-sm:          0.7rem;
+$input-padding-x-sm:          1.2rem;
+$input-border-radius:         3px;
```
Comment (mmexvr) [inline src/styles/_variables.scss]: Do we need a specific border-radius for input, can't we use the border-radius variable?

File: src/styles/components/_forms.scss [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  }
+
+  &:-ms-input-placeholder,
+  &::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+    color: $dark-01;
+    opacity: 1; /* Firefox */
+  }
+  
+  // &:-ms-input-placeholder { /* Internet Explorer 10-11 & Edge */
+  //   color: $dark-01;
```
Comment (mmexvr) [inline src/styles/components/_forms.scss]: Can probebly remove this for now, since we don't support anything with IE10 and we have the same rule above

**Review by mmexvr — CHANGES_REQUESTED** [review]

File: src/styles/_variables.scss [hunk 3]
```diff
@@ -110,9 +114,19 @@ $h4-font-size-article:        $font-size-article * 1.33;
 $h5-font-size-article:        $font-size-article;
 $h6-font-size-article:        $font-size-article;
 
+$border-color:                $light;
+$table-border-color:          $border-color;
+$input-border-color:          $border-color;
+
+$input-padding-x:             1.2rem;
```
Comment (mmexvr) [inline src/styles/_variables.scss:129]: Think this is more in taste, but can't we have one rule for padding? instead of two for sizing?

**Review by mmexvr — CHANGES_REQUESTED** [review]

**Comment by mmexvr** [comment]
Conflict

File: src/styles/_variables.scss (hunk 1 above)
Comment (helloimela) [reply src/styles/_variables.scss]: I will add global border-radius after I merge master to this branch

File: src/styles/components/_forms.scss (hunk 2 above)
Comment (helloimela) [reply src/styles/components/_forms.scss]: Okay, tested this, unfortunately need to keep it separated to make it work :(

File: src/styles/_variables.scss (hunk 3 above)
Comment (helloimela) [reply src/styles/_variables.scss:129]: So, to keep the height of the input according to the visual guidelines we need to do either one of these:
1. Add different padding
Or
2. Set fix height on the input (a solution we had in CUI 3).
I prefer solution 1 though, feel like we should avoid fixed height on element.

**Review by mmexvr — APPROVED** [review]

**Pull request merged**